conf/modules/qc_bam.config

```diff
@@ -53,6 +53,16 @@ process {
         ext.prefix = { "${meta.id}_mosdepth" }
     }
 
+    withName: '.*QC_BAM:NGSBITS_SAMPLEGENDER' {
+        //  NGSBITS_SAMPLEGENDER needs a chrX and chrY in order to run so we skip it for the two test profiles
+        ext.when = { !(workflow.profile.tokenize(',').intersect(['test', 'test_one_sample', 'test_sentieon']).size() >= 1) || workflow.stubRun }
```

Comment:
Let's add a prefix to the output file for clarity. Could be something like:
```
ext.prefix = { "${meta.id}_sex }
```
or 
```
ext.prefix = { "${meta.id}_ngsbits_sex }
```

Reply:
Done! :)